gatherers/defs.go

```diff
@@ -16,7 +16,13 @@ type GatheredFact struct {
 	Category FactCategory `json:"category,omitempty"`
 }
 
+const (
+	Static string = "static"
```

Comment:
I'm wondering if a quick refactoring of these consts might be useful - 
Perhaps instead of "Static" and "Dynamic" - you could give it a little more descriptive name, or indicate that it's a const by calling it something like
GATHERER_TYPE_STATIC and GATHERER_TYPE_DYNAMIC?